server/src/testE2E/java/com/flipkart/varadhi/DefaultAuthZProviderTests.java

```diff
@@ -0,0 +1,327 @@
+package com.flipkart.varadhi;
```

Comment:
add integration with existing org, team, etc crud flows (as other PR)
all varadhi apis are authorized if enabled

Reply:
to allow authn to work either go the token way or impl default/fake authn provider to take identity from headers being passed.